brave-clean-slate.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# This script removes Brave browser configuration, bookmarks, extensions and residual files.
+# Usage: ./brave-clean-slate.sh
+
+sudo rm -rf /Library/Application\ Support/BraveSoftware
+rm -rf ~/Library/Application\ Support/BraveSoftware
```

Comment:
The Windows version has a flag `--delete_profiles`. Instead of adding the new file `brave-clean-slate.sh`, could you add a flag `--delete_profiles` to `uninstall-brave.sh`? It should do `rm -rf ~/Library/Application\ Support/BraveSoftware`.